src/vm/i386/excepcpu.h

```diff
@@ -90,15 +91,24 @@ extern VOID ResetSEHRecord(PEXCEPTION_REGISTRATION_RECORD record);
 
 #endif
 
+
+PEXCEPTION_REGISTRATION_RECORD GetCurrentSEHRecord();
```

Comment:
GetCurrentSEHRecord etc. is a part of the change that was made to compile with the WIN32EXCEPTIONS and that should be reverted now (it was made in https://github.com/dotnet/coreclr/pull/8613)